src/components/Quote.tsx

```diff
@@ -29,9 +29,11 @@ const QuoteContainer = styled.blockquote`
 
   ::after {
     content: '';
+    display: none;
```

Comment:
If we set `display: none` there is no need to have `content: ''`, it bears no value.